### PR TITLE
zweinroth HW01

### PR DESCRIPTION
**Pull request by zweinroth**
The overall process was relatively simple.  The instructions on the homework were clear and easy to follow. I did struggle with the formatting at certain points, but the 'mastering markdown' page on GitHub was a great reference for figuring out how to upload the photos, type in the list/chart, and create different headers for the assignment.  Specifically for the image upload, I was struggling trying to create the right path for the code to upload from, but then found that placing the images into the same working directive/homework folder allowed me to simply type in the .jpg file name without having to orchestrate an extensive path.  The 10 different commits are pretty easy to follow, the first 6-7 submissions were simply to make sure that I forked the file, created the repository, formatted the markdown to my satisfaction, and was saved the .md correctly.  The final submissions were all different small edits to ensure flow and clarity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+![Image of TREX](TREX.JPG)
+
+He loved this costume and knew that this would be the __*greatest* outfit that he would ever wear__. Unfortunately, his inability to even smile for the camera makes him seem sad and unamused by the pageantry of the holiday.
+Luckily for Zach, he grew out of that phase and became a happy child, who played outside with the neighborhood kids, competed in athletics, and excelled in school.
+
+### Adulthood
+
+As Zach now enters his adulthood, he is amazed by the opportunities the world presents him.  He is at an amazing university, where he can take classes that both _intrigue_ and _challenge_ him. A prime example would be the class he is taking currently, __Computing for the Social Sciences__, where he can learn about the intracacies of the core principles of programming and then apply it to his future research and professional career. There are many more things to come in his life, but he knows that all he should do now is hunker down and get the most out of his last year at the University of Chicago.
```
Comment (Tracylyy) [inline README.md:32]: Good use of italic and bold.

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
-# hw01
\ No newline at end of file
+## Zachary Weinroth
+## October 7, 2019
+## HW01
+
+### The Beginning
+Zach Weinroth was born May 8, 1998 in Summit County, New Jersey.  His parents feared the effects of their child growing up in Jersey and sought out career options in other places. They were able to find work in Denver, Colorado, but they had to weigh the options against each other to be sure it was the right place for the Weinroth family.
```
Comment (Tracylyy) [inline README.md:6]: I really like how you used headers! It looks like pretty organized.

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
...
+
+![Image of Heimlich](Heimlich.JPG)
+
+__*OOOHHH?*__
+
+### Childhood
+Zachary's childhood was full of wonder, excitement, and curiosity. The only problem, was that he was never good at expressing these emotions facially.  For example, here is Zach on Halloween when he was 3 years old:
+
+![Image of TREX](TREX.JPG)
+
```
Comment (Tracylyy) [inline README.md:26]: Actually I can't see the image. Maybe you did not put the image in the same folder, but I am not sure if it is the reason.

**Review by medcar8879 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Your biography is super interesting and actually made me laugh out loud. Thanks for that! Your headers organized everything well, so it was easy to follow. Very impressive table! Did you choose to make the rows alternate colors? If so, very impressive! I have no suggestions for improvements, great job!

File: README.md [hunk 4]
```diff
@@ -1 +1,32 @@
-# hw01
\ No newline at end of file
+## Zachary Weinroth
+## October 7, 2019
+## HW01
+
+### The Beginning
```
Comment (jsimsspeyer) [inline README.md:5]: Good use of different header types

File: README.md [hunk 5]
```diff
@@ -1 +1,32 @@
...
+Zach Weinroth was born May 8, 1998 in Summit County, New Jersey.  His parents feared the effects of their child growing up in Jersey and sought out career options in other places. They were able to find work in Denver, Colorado, but they had to weigh the options against each other to be sure it was the right place for the Weinroth family.
+
+New Jersey | Colorado
```
Comment (jsimsspeyer) [inline README.md:8]: The chart formatted nicely. Good work!

File: README.md (hunk 1 above)
Comment (jsimsspeyer) [inline README.md:32]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     *      |              |            |
| **Coding strategy**         |    *       |              |            |
| **Presentation**            |      *     |              |            |
| **Achievement, creativity** |     *      |              |            |
| **Ease of access**          |      *     |              |            |

Remarks:

*You went above and beyond with the chart
*Demonstrated mastery of italics, bold, headers, and images 
*Had clear instructions regarding changes to the READ.ME and challenges

File: README.md [hunk 6]
```diff
@@ -1 +1,32 @@
...
+---------- | --------
+Close to New York City | The center of the country
+Close to the Atlantic Ocean | Last of NA lands to be submerged by rising sea levels
+... | Nature is everywhere
+... | Land is cheap and spacious, room for kids to play
+It's *New Jersey* | It's **Colorado**
+
+The list made it clear that Denver was the best place for the family and that New Jersey didn't have enough positives to stay:
+
+![Image of Heimlich](Heimlich.JPG)
```
Comment (rrodriguezbarron) [inline README.md:18]: Both images didn't load correctly when I reviewed the assignment. Review the markdown for inserting images.

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |           |     x         |            |

Remarks:

* You definitely have a good grasp of how to program. The table was a *very* good extra touch and I commend you for it.
* I couldn't see the images in the markdown file.

**Pull request closed without merging**